### PR TITLE
Feature/details screen

### PR DESCRIPTION
**Pull request by DejanuTeodora**
## Description
Added description, skills, projects, level on details screen

## Related
https://trello.com/c/WO0QNiyJ

## Steps to Test or Reproduce

1. Install app
2. Open app
3. Tap on freak

### CODE REVIEW

**Comment by DejanuTeodora** [comment]
[iOS - Details Screen - Setup UI](https://trello.com/c/WO0QNiyJ/85-ios-details-screen-setup-ui)

File: Shared/Features/FreakDetails/View/FreakDetailsView.swift [hunk 1]
```diff
@@ -3,21 +3,65 @@ import SwiftUI
 struct FreakDetailsView: View {
     @Environment(\.presentationMode) var presentationMode
     var viewModel: FreakDetailsViewModel
+    // let freak = Bundle.main.decode([Freak].self, from: "MockData.json")
```
Comment (vladstanescu94) [inline Shared/Features/FreakDetails/View/FreakDetailsView.swift:6]: Remove comments, please

File: Shared/Features/FreakDetails/View/FreakDetailsView.swift [hunk 2]
```diff
@@ -3,21 +3,65 @@ import SwiftUI
...
+                    .navigationBarBackButtonHidden(true)
+                    .navigationBarItems(leading: Button(action: { self.presentationMode.wrappedValue.dismiss() }) {
+                        HStack {
+                            Image(systemName: "chevron.left")
+                            Text(viewModel.fullName)
+                        }
+                    })
+
+                Text(viewModel.description)
+                    .padding()
```
Comment (vladstanescu94) [inline Shared/Features/FreakDetails/View/FreakDetailsView.swift:33]: We need to make the text elements full width, to consider the case where we have fewer words on the description. Don't rely on the content to layout the view.

File: Shared/Features/FreakDetails/View/FreakDetailsView.swift [hunk 3]
```diff
@@ -3,21 +3,65 @@ import SwiftUI
...
+
+            .frame(maxWidth: 350, maxHeight: 650, alignment: .top)
+        }
     }
 }
 
 struct FreakDetailsView_Previews: PreviewProvider {
     static var previews: some View {
         FreakDetailsView(viewModel: FreakDetailsViewModel(freak: Freak(imageName: "test1", firstName: "Vlad", lastName: "Stanescu", description: "Lorem ipsum", skills: ["iOS"], projects: ["Epix"], role: "iOS Developer", level: "Senior", norm: "Fulltime", id: 0)))
+            .previewInterfaceOrientation(.portraitUpsideDown)
```
Comment (vladstanescu94) [inline Shared/Features/FreakDetails/View/FreakDetailsView.swift:65]: Remove this preview orientation

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]
Also, use the spacing and the sizes from the design if possible.

**Review by vladstanescu94 — APPROVED** [review]

**Review by rmoldo — APPROVED** [review]

**Pull request merged**